Loop/Views/SettingsView.swift

```diff
@@ -193,6 +193,7 @@ public struct SettingsView_Previews: PreviewProvider {
                                           pumpManagerSettingsViewModel: DeviceViewModel(),
                                           cgmManagerSettingsViewModel: DeviceViewModel(),
                                           therapySettings: TherapySettings(),
+                                          supportedBasalRates: nil,
```

Comment:
Would we want to get this from the pumpManager?

Reply:
Yes, stay tuned :)